.github/ISSUE_TEMPLATE/approved-reviewers-request.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Fed. engineer PR approver checklist
+about: Checklist to ensure you are prepared to become a PR approver for your team's code
+title: ''
+labels: ['']
+assignees: ''
+---
+# PR Approver Preparation Checklist
+
+Certain federal engineers can get approval to review their team's code and bypass a Platform PR review. Use this checklist to ensure you are prepared to become a PR approver for your team's code.
+
+**Engineer Name:**
+
+**Team Name & [GitHub Team](https://github.com/orgs/department-of-veterans-affairs/teams):**
+<!-- Name of your team and code-owning GitHub team(s) -->
+
+## Training
+
+- [ ] Thoroughly review the [Code review guide for vets-api](https://vfs.atlassian.net/wiki/spaces/BCP/pages/3155394563/Code+review+guide+for+vets-api).
```

Comment:
QUESTION: Is this only for the API or does this reviewer also include the website as well?

Reply:
The intent will be the other repos as well.

Reply:
@mdewey  good question. I tried to make this template as generic as possible so it could be for the frontend _and_ backend, but I don't think we're quite there yet. i.e. there's no code review guide for vets website.